### ISSUE TITLE
A simple way of quickly running benchmarks

### ISSUE THREAD

**Issue by ltratt** [softdevteam/lua_benchmarking#23]
I think we need to provide a simple way of people who don't have Krun to run benchmarks quickly, using whatever Lua VM they have knocking around, to encourage people to contribute to this repo. At its most basic it should just be something like:

```sh
$ lua quick_run.lua
Running bench1: .........................
  Mean: 2.2s +/- 0.1s
Running bench2: .........................
  Mean: 0.8s +/- 0.05s
```

What we need to do is a get a reasonable trade-off between ease of use and not being completely statistically misleading. My suggestion is that we run each benchmark for 30 in-process iterations (printing a "." to stdout as each is being run, so the user knows something is happening), and after each printing a mean and a 95% confidence interval. `quick_run.lua` could take an optional integer parameter which varies the number of in-process iterations (so `quick_run.lua 10` would run 10 in-process iterations).

We would of course need to put a warning in the README along the lines of "this isn't statistically reliable, but it is better than nothing." I can handle that part if you want me to.

**Comment by fsfod** [comment]
simplerunner.lua does about half this already

**Comment by ltratt** [comment]
Aha, that's a good start :) Let's get it doing the rest outlined above and then we're sorted!

**Comment by fsfod** [comment]
> whatever Lua VM they have knocking around
I think it will be pretty limiting to depend on plain Lua and not some build of LuaJIT with the ffi system i can callout in to the os and collect better data.

**Comment by ltratt** [comment]
Can one not install the FFI with plain Lua?

**Comment by fsfod** [comment]
That would be a binary module i was trying avoid to having to include any

**Comment by ltratt** [comment]
Is building it difficult? If so, we could presumably say "if you want to run this on plain Lua you will need to have installed XXX yourself"?

**Comment by ltratt** [comment]
#26 mostly addressed this, but didn't add any documentation to the README. @tfransham can you add a few lines to the README that tell people how to run things?

**Comment by ltratt** [comment]
I've started looking into fixing this, so the action is now on me.

**Comment by ltratt** [comment]
Fixed in #28.
